configuration/pih/scripts/global/liberia.js

```diff
@@ -1,26 +1,14 @@
 
 
 var omrs = {
+
     toggleExposureDurationField: function (x, containerId) {
         if (x == 1) {
             $('#' + containerId).show();
         } else if (x == 0) {
             $('#' + containerId).hide();
         }
```

Comment:
I think the best thing to do for now is to remove all of the Javascript below that was originally added (seemingly) to manipulate the conditions widget on the ncd and mental health forms.  If there are use cases that this was solving which can be identified and described, we can re-ticket to add back in the desired functionality.  Currently, it is not clear what (if anything) this is doing on the current forms, and it is causing JS errors on the console on all pages in the system outside of HFE edit/view mode, so I think it's best to remove for now.  @lnball ?